examples/basics/projects.ipynb

```diff
@@ -48,7 +48,7 @@
     {
```

Comment:
<p>Can we add some text here about why are we creating ontology and labels, and re-format some of the code blocks like we discussed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1343/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Hey! I made these changes. Let me know what you think.